Maintaining.md

```diff
@@ -4,6 +4,7 @@
 
 - Make your changes
 - Update `version` value on `version.rb` file
+- To install the package from local codes, you should build a gem `gem build veritrans.gemspec` after that `gem install veritrans-2.4.0.gem`
```

Comment:
To avoid hard coding version number, please update to:
> ... `gem install veritrans-x.y.z.gem` (replace `x.y.z` with the version number e.g. `gem install veritrans-2.4.0.gem`)

Reply:
Also: after installing local gem, if you make code changes on the library code, does it automatically update the locally installed gem? or you need to re-build & reinstall gem each time?

Reply:
need to rebuild and reinstall kak

Reply:
Then please add that note to the file @Andrenzo17 . That it need to be rebuild&install to apply code changes. Something like:

- To install the package from local codes, you should build a gem `gem build veritrans.gemspec` after that `gem install veritrans-x.y.z.gem` (replace `x.y.z` with the version number e.g. `gem install veritrans-2.4.0.gem`)
  - Note: In case you want to test another new code changes, you will need to re-build & re-install, so that your latest code will be used by the locally installed gem